test/test_fx.py

```diff
@@ -604,5 +606,35 @@ def test_construct_root_dict(self):
         self.assertEqual(out, ref_out)
 
 
+    def test_get_all_users_of(self):
+        graph : torch.fx.Graph = torch.fx.Graph()
+        a : torch.fx.Node = graph.create_node('placeholder', 'x')
+        b : torch.fx.Node = graph.create_node('call_module', 'foo.bar.baz', args=(a,))
+        c : torch.fx.Node = graph.create_node('get_attr', 'zip.zap.zam')
+        d : torch.fx.Node = graph.create_node('call_function', operator.add, args=(b, c))
+        graph.output(d)
+
+        linear_mod : torch.nn.Module = torch.nn.Linear(3, 4)
+        add_param : torch.Tensor = torch.rand(3, 4)
+        gm : torch.fx.GraphModule = torch.fx.GraphModule(
+            {'foo.bar.baz': linear_mod, 'zip.zap.zam' : add_param}, graph)
+
+        for index, node in enumerate(graph.nodes):
+            user_indexes = GraphManipulation.get_all_users_of(index, gm)
+            for i in user_indexes:
+                n = graph.nodes[i]
+                args = map_arg(n.args, lambda n: n)
+                kwargs = map_arg(node.kwargs, lambda n: n)
+                assert isinstance(args, tuple)
+                assert isinstance(kwargs, dict)
+                kwargs_nodes = tuple([v for k, v in kwargs.items()])
+                assert node in args or kwargs_nodes
```

Comment:
Similarly to the comment in the implementation, instead of using `map_arg` here I think you should traverse the `args` and `kwargs` data structures to confirm the def-edge